### DISCUSSION TITLE
Best Way To Disable/Remove unused sensors

### DISCUSSION THREAD

**Discussion by thenoid** [emporia-vue-local/esphome#262] [Q&A]
What's best way to disable sensors that aren't in use?  EG  12-16 aren't used in one of my subpanels.  Is commenting out in the YAML the best way?  is there a flag to add?

Promise i searched the discussions for "remove" and "disable" and didn't find something.

**Comment by flaviut** [comment]
Commenting them out or erasing the lines is the right way to do this. Don't
forget to update the places where they are referenced as well.
